kibbeh/src/lib/tests/constants.test.ts

```diff
@@ -0,0 +1,25 @@
+import { linkRegex } from "../constants";
+
+test("Link Regex Test 1(Only Link)", () => {
```

Comment:
checkout `describe` to group all the tests together

Reply:
Done.